### PR TITLE
Allow hiding/showing the obs tree

### PR DESCRIPTION
**Pull request by cquiroz**
https://github.com/gemini-hlsw/explore/assets/3615303/24685a05-1cc2-48b7-a04d-3076d9a22d5e

We may want to make this persistent later on

### CODE REVIEW

File: explore/src/main/scala/explore/config/SpectroscopyModesTable.scala [hunk 1]
```diff
@@ -625,7 +625,10 @@ private object SpectroscopyModesTable extends TableHooks:
                 for
                   virtualizer <- refOpt
                   idx         <- selectedIndex.value
-                yield virtualizer.scrollToIndex(idx + 1, ScrollOptions)
+                yield {
+                  println(s"Scroll to  $idx")
```
Comment (hugo-vrijswijk) [inline explore/src/main/scala/explore/config/SpectroscopyModesTable.scala:629]: Println?

Comment (cquiroz) [reply explore/src/main/scala/explore/config/SpectroscopyModesTable.scala:629]: Ups

File: explore/src/main/scala/explore/tabs/ObsTabContents.scala [hunk 2]
```diff
@@ -67,11 +69,22 @@ import react.draggable.Axis
 import react.gridlayout.*
 import react.hotkeys.*
 import react.hotkeys.hooks.*
+import react.primereact.Button
 import react.resizeDetector.*
 import react.resizeDetector.hooks.*
 
 import scala.concurrent.duration.*
 
+object DeckShown extends NewType[Boolean]:
```
Comment (toddburnside) [inline explore/src/main/scala/explore/tabs/ObsTabContents.scala:78]: What does `Deck` mean in this context?

**Review by toddburnside — APPROVED** [review]
LGTM

File: explore/src/main/scala/explore/tabs/ObsTabContents.scala (hunk 2 above)
Comment (cquiroz) [reply explore/src/main/scala/explore/tabs/ObsTabContents.scala:78]: Well the obs tree area

**Pull request merged**